client/src/components/atoms/InlineComment.vue

```diff
@@ -0,0 +1,48 @@
+<template>
```

Comment:
At a quick glance, it looks likes this isn't referenced anywhere anymore. delete?